milestone2.ts

```diff
@@ -92,23 +286,45 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+//type GetCity = (data: Volunteer[]) => string;
+
+let findIndexAns: GetNumber = (data) => {
+  let i = 0;
+  let j = data[i].city;
+  
+  while((j !== "SF" && i !== data.length -1)){
+    i++; 
+    j = data[i].city;
+  }
+  if(j === "SF"){
+    return i; 
+  }else{
+    return -1; 
+  }
```

Comment:
FYI there is a findIndex function in TS that makes this function a lot easier to write!